packages/svelte/types/index.d.ts

```diff
@@ -2215,7 +2218,7 @@ declare module 'svelte/transition' {
 	 * https://svelte.dev/docs/svelte-transition#crossfade
 	 * */
 	export function crossfade({ fallback, ...defaults }: CrossfadeParams & {
-		fallback?: (node: Element, params: CrossfadeParams, intro: boolean) => TransitionConfig;
+		fallback?: ((node: Element, params: CrossfadeParams, intro: boolean) => TransitionConfig) | undefined;
```

Comment:
didn't update dts-buddy, @Rich-Harris why did this change? isn't the previous version more correct if you're using the "strict property types" option? are we just not using that in the compiler options and it's not available in strict mode by default? need to check

Reply:
Errr I have no idea. Seems like a TypeScript thing (nothing changed in dts-buddy that would cause that). Seems this is just undoing this recent change: https://github.com/sveltejs/svelte/commit/8cbe8c1197011d69198a0e4816f2198e60b83147#diff-b1326f5f98417cd282e68bc1e4e6b869e69156c42a36afd2fd0721927000bd36R2218